benchmarks/distributed/rpc/parameter_server/configurations/server_configurations.json

```diff
@@ -0,0 +1,14 @@
+{
+    "AverageCpuParameterServer": {
+        "ps_class": "AverageParameterServer",
+        "configurations": {
+            "use_cuda_rpc": false
+        }
+    },
+    "AverageCudaParameterServer": {
```

Comment:
curious, why do we need this config file. Is it possible to directly use `AverageCudaParameterServer` class when users pass in `--server=AverageCudaParameterServer`. Is there any reason that we need to use this config as a relay?

Reply:
If I remove this config file, then the extensible quality attribute is affected. All parameter servers will have the same arguments given in the initialization of the class. The server class can be passed as a benchmark argument, and the config could be the arguments for the server.

Reply:
actually, I can remove the config file but keep the functionality in the launcher. a user needs to specify a config file for additional arguments for their server or trainer.